### DISCUSSION TITLE
Example Idea Submission (Keep it short & simple)

### DISCUSSION THREAD

**Discussion by pandeyanujk** [memser-spaceport/pl-network-hub#2] [Announcements]
**PL Network - It's time to add your ideas & feature requests so we can all discuss & focus our efforts on the highest-value items.** 

### What is the idea? 
_please describe your idea_

### Why is this important?
_please make a case for this idea. What value would this add to the network? Would something become better or easier if this was implemented?_

**Comment by AlexxNica** [comment]
@pandeyanujk FYI: labels are limited to users with "Triage" permissions or higher

More info: https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role
